projects/plugins/jetpack/extensions/blocks/ai-assistant/lib/prompt/index.ts

```diff
@@ -44,50 +44,95 @@ export type PromptItemProps = {
  * @param {string} options.context - The context of the prompt.
- * @param {Array<string>} options.rules - The rules to follow.
- * @returns {PromptItemProps} The initial system prompt.
+ * @returns {PromptItemProps}      - The initial system prompt.
  */
 export function getInitialSystemPrompt( {
-	context = 'You are an AI assistant, your task is to generate and modify content based on user requests. This functionality is integrated into the Jetpack product developed by Automattic. Users interact with you through a Gutenberg block, you are inside the Wordpress editor',
-	rules,
+	context = 'You are a ghostwriter. Your task is to generate and modify content based on user requests. This functionality is integrated into the Jetpack product developed by Automattic. Users interact with you through a Gutenberg block, you are inside the Wordpress editor',
 }: {
 	context?: string;
+} = {} ): PromptItemProps {
+	return { role: 'system', content: context };
+}
+
+/**
+ * Helper function to get the rules user prompt.
+ *
+ * @param {object} options                - The options for the prompt.
+ * @param {Array<string>} options.rules   - The rules to follow.
+ * @param {boolean} options.isTitlePrompt - Whether the the prompt refers to a post title.
+ * @returns {PromptItemProps}             - The rules user prompt.
+ */
+export function getRulesPrompt( {
+	rules = [],
+	isTitlePrompt = false,
+}: {
 	rules?: Array< string >;
+	isTitlePrompt?: boolean;
 } ): PromptItemProps {
 	// Rules
-	let extraRules = '';
-	if ( rules?.length ) {
-		extraRules = rules.map( rule => `- ${ rule }.` ).join( '\n' ) + '\n';
+	const promptRules = [
+		...rules,
+		'Execute the request without any acknowledgement of it, only outputting the final result',
+		'Avoid sensitive or controversial topics and ensure your responses are grammatically correct and coherent',
+		'If you cannot generate a meaningful response to a user\'s request, reply with "__JETPACK_AI_ERROR__". This term should only be used in this context, it is used to generate user facing errors',
+	];
+	if ( ! isTitlePrompt ) {
+		promptRules.push( 'Format your responses in Markdown syntax, ready to be published' );
 	}
-	const prompt = `${ context }.
-Strictly follow these rules:
 
-${ extraRules }- Format your responses in Markdown syntax, ready to be published.
-- Execute the request without any acknowledgement to the user.
-- Avoid sensitive or controversial topics and ensure your responses are grammatically correct and coherent.
-- If you cannot generate a meaningful response to a user’s request, reply with “__JETPACK_AI_ERROR__“. This term should only be used in this context, it is used to generate user facing errors.
-`;
+	let formattedRules = '';
+	formattedRules = promptRules.map( rule => `- ${ rule }.` ).join( '\n' );
+
+	const prompt = `Strictly follow these rules:
+${ formattedRules }`;
 
-	return { role: 'system', content: prompt };
+	return { role: 'user', content: prompt };
```

Comment:
What's the motivation behind replacing the role here? These rules do seem to belong to the `system` role.

<img width="914" alt="image" src="https://github.com/Automattic/jetpack/assets/77539/5a89503f-eabd-4cad-bed1-51bd536b47c7">

Reply:
It seems to respect the rules better if they are in a `user` message. Did you notice any change?

Reply:
I dont :-(